puppeteer/bundleRequests.test.js

```diff
@@ -0,0 +1,65 @@
+import puppeteer from 'puppeteer';
+import cypressConfig from '../cypress/support/config';
+
+let browser;
+let page;
+let requests = [];
+
+const config = cypressConfig('local');
+
+const services = [
+  { service: 'news', path: `/news/articles/${config.assets.news}` },
+  { service: 'persian', path: `/persian/articles/${config.assets.persian}` },
+  { service: 'igbo', path: '/igbo' },
+  { service: 'yoruba', path: '/yoruba' },
+  { service: 'pidgin', path: '/pidgin' },
+];
+
+describe('Js bundle requests', () => {
+  beforeEach(async () => {
+    browser = await puppeteer.launch();
+    page = await browser.newPage();
+
+    page.on('request', request => {
+      requests.push(request.url());
+    });
+  });
+
+  afterEach(async () => {
+    browser.close();
+    requests = [];
+  });
+
+  services.forEach(({ service, path }) => {
+    describe(service, () => {
+      beforeEach(async () => {
+        await page.goto(`http://localhost:7080${path}`, {
+          waitUntil: 'networkidle2',
```

Comment:
Why did you pick `networkidle2` instead of `networkidle0`?

https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagegotourl-options

Reply:
Arbitrarily really, seemed safer to wait a bit longer just incase, v unlikely but still